app/src/main/java/com/framgia/awesomecity/screen/orders/OrderViewModel.java

```
@@ -0,0 +1,40 @@
+package com.framgia.awesomecity.screen.orders;
+
+import android.databinding.BaseObservable;
+import android.databinding.Bindable;
+
+import com.framgia.awesomecity.data.model.Order;
+
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+
+/**
+ * Created by lamvu on 8/9/2017.
+ */
+
+public class OrderViewModel extends BaseObservable {
+
+    @Bindable
+    private Order mOrder;
+
+    public OrderViewModel(Order order){
+        mOrder = order;
+    }
+
+    public Order getOrder(){
+        return mOrder;
+    }
+
+    @Bindable
+    public String getWaitTime(){
+        Calendar calendar = Calendar.getInstance();
+        int minutes = calendar.get(Calendar.MINUTE);
+        return minutes + " minutes";
+    }
+
+    private long convertTimeToMinutes(String updatedAt){
+        DateFormat format = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'");
```

WARNING: [SimpleDateFormat] To get local formatting use `getDateInstance()`, `getDateTimeInstance()`, or `getTimeInstance()`, or use `new SimpleDateFormat(String template, Locale locale)` with for example `Locale.US` for ASCII dates.
 Almost all callers should use `getDateInstance()`, `getDateTimeInstance()`, or `getTimeInstance()` to get a ready-made instance of SimpleDateFormat suitable for the user's locale. The main reason you'd create an instance this class directly is because you need to format/parse a specific machine-readable format, in which case you almost certainly want to explicitly ask for US to ensure that you get ASCII digits (rather than, say, Arabic digits).

Therefore, you should either use the form of the SimpleDateFormat constructor where you pass in an explicit locale, such as Locale.US, or use one of the get instance methods, or suppress this error if really know what you are doing.